include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -26,10 +26,14 @@
 #endif
 
 #ifndef INDEXSTOREDB_PUBLIC
-# if defined (_MSC_VER)
-#  define INDEXSTOREDB_PUBLIC __declspec(dllimport)
+# if defined(_WIN32) && defined(_DLL)
+#   if defined(CIndexStoreDB_EXPORTS)
```

Comment:
Is anything setting `CIndexStoreDB_EXPORTS` in swiftpm?

Reply:
No, that should be set when compiling the sources for it.  CMake does that implicitly.  I’ll upload a separate patch for that.